src/routes/hooks/hooks-cache.spec.ts

```diff
@@ -98,158 +103,6 @@ describe.skip('Post Hook Events for Cache (Unit)', () => {
-      address: faker.finance.ethereumAddress(),
-      messageHash: faker.string.hexadecimal({ length: 32 }),
-    },
-    {
-      type: 'CHAIN_UPDATE',
-    },
-    {
-      type: 'REORG_DETECTED',
-      chainId: faker.string.numeric(),
-      blockNumber: faker.number.int(),
-    },
-    {
-      type: 'SAFE_APPS_UPDATE',
-    },
-    {
-      type: 'SAFE_CREATED',
-      address: faker.finance.ethereumAddress(),
-      blockNumber: faker.number.int(),
-    },
-  ])('accepts $type', async (payload) => {
-    const chainId = faker.string.numeric();
-    const data = {
-      chainId: chainId,
-      ...payload,
-    };
-    networkService.get.mockImplementation(({ url }) => {
-      switch (url) {
-        case `${safeConfigUrl}/api/v1/chains/${chainId}`:
-          return Promise.resolve({
-            data: rawify(chainBuilder().with('chainId', chainId).build()),
-            status: 200,
-          });
-        default:
-          return Promise.reject(new Error(`Could not match ${url}`));
-      }
-    });
-
-    await request(app.getHttpServer())
-      .post(`/hooks/events`)
-      .set('Authorization', `Basic ${authToken}`)
-      .send(data)
-      .expect(202);
-  });
-
-  it('returns 400 (Bad Request) on unknown payload', async () => {
-    const data = {
-      type: 'SOME_TEST_TYPE_THAT_WE_DO_NOT_SUPPORT',
-      safeTxHash: 'some-safe-tx-hash',
-    };
-    networkService.get.mockImplementation(({ url }) => {
-      switch (url) {
-        case `${safeConfigUrl}/api/v1/chains/1`:
-          return Promise.resolve({
-            data: rawify(chainBuilder().with('chainId', '1').build()),
-            status: 200,
-          });
-        default:
-          return Promise.reject(new Error(`Could not match ${url}`));
-      }
-    });
-
-    await request(app.getHttpServer())
-      .post(`/hooks/events`)
-      .set('Authorization', `Basic ${authToken}`)
-      .send(data)
-      .expect(422)
-      .expect({
-        statusCode: 422,
-        code: 'invalid_union_discriminator',
-        options: [
-          'CHAIN_UPDATE',
-          'DELETED_MULTISIG_TRANSACTION',
-          'EXECUTED_MULTISIG_TRANSACTION',
-          'INCOMING_ETHER',
-          'INCOMING_TOKEN',
-          'MESSAGE_CREATED',
-          'MODULE_TRANSACTION',
-          'NEW_CONFIRMATION',
-          'MESSAGE_CONFIRMATION',
-          'OUTGOING_ETHER',
-          'OUTGOING_TOKEN',
-          'PENDING_MULTISIG_TRANSACTION',
-          'REORG_DETECTED',
-          'SAFE_APPS_UPDATE',
-          'SAFE_CREATED',
-        ],
-        path: ['type'],
-        message:
-          "Invalid discriminator value. Expected 'CHAIN_UPDATE' | 'DELETED_MULTISIG_TRANSACTION' | 'EXECUTED_MULTISIG_TRANSACTION' | 'INCOMING_ETHER' | 'INCOMING_TOKEN' | 'MESSAGE_CREATED' | 'MODULE_TRANSACTION' | 'NEW_CONFIRMATION' | 'MESSAGE_CONFIRMATION' | 'OUTGOING_ETHER' | 'OUTGOING_TOKEN' | 'PENDING_MULTISIG_TRANSACTION' | 'REORG_DETECTED' | 'SAFE_APPS_UPDATE' | 'SAFE_CREATED'",
-      });
-  });
```

Comment:
These were removed as we receive no response. If we want to check this, however, we could check [whether an error is logged](https://github.com/safe-global/safe-client-gateway/blob/unskip-cache-tests/src/domain/hooks/hooks.repository.ts#L43).

Reply:
I think we just need to add a new test to verify that an error is thrown when the event name is valid but is not whitelisted.